### ISSUE TITLE
Function to_timestamp without format does not work in v 7

### ISSUE THREAD

**Issue by ideoma** [questdb/questdb#3090]
### Describe the bug

This used to work in 6.5.3

```sql
select to_timestamp('2023-03-03 10:36:11.228932+00:00') 
```

but in 7.0.1 return `null`

### Environment

```markdown
- **QuestDB version**:7.0.1
```

**Comment by jerrinot** [comment]
that looks like an intentional change: https://github.com/questdb/questdb/pull/2557
> change to_timestamp(S) to always interpret arg as micros. Similar function in PG interprets string as double, where scale values used for millis or micros and integers as seconds. I feel our approach is clearer

Indeed `select to_timestamp('500000000000000');` returns `1985-11-05T00:53:20.000000Z`
Still, we should print an error when the string is not a number.

**Comment by bluestreak01** [comment]
this issue refers to ' ' (space) delimiter no longer working, usually this requires 'T'

**Comment by ideoma** [comment]
```SQL
cast('2023-03-03 10:36:11.228932+00:00' as timestamp)
```

works on 7.0.1, no `T` needed

**Comment by ideoma** [comment]
`to_timestamp` with a single parameter is not documented. It used to work one way, but now it's not working. 
Would be good to make it compatible with the old version or with Postgres and document it.

**Comment by ideoma** [comment]
I don't agree. I read this paragraph so that call `to_timestamp` is optional. So that you can do 

```SQL
timestamp > '2023-03-03 10:36:11.228932+00:00'
```

I don't think we say that the `format` parameter is optional anywhere.

**Comment by jerrinot** [comment]
good point, I misinterpreted the paragraph, my bad.
